AUTHORS

```diff
@@ -87,6 +87,7 @@ Damian Skrzypczak
 Daniel Grana
 Daniel Hahler
 Daniel Nuri
+Daniel Sánchez Castelló
```

Comment:
Why are you adding this many people? Usually PR authors just add themselves. Not sure it is OK to add other people without their consent.

Reply:
Hi @nicoddemus, this PR is part of a mob-programming session at @clarityai-eng.

The people involved in the thinking and writing of this change are:
@toclarity - Tatiana
@Wolfremium13 - Kevin
@victorrmclarity - Victor
@DaniSancas - Daniel

Is it correct to write all our names in the AUTHORS section or should we delete some of them?

Reply:
Ahh no worries then, thanks for the clarification.

Reply:
Yes, everyone on the list was in the session and gave their consent.

It's the first time we contribute to this project. Are there any next steps to be done?

Thank you!

Reply:
No that's it!

Thanks @toclarity, @Wolfremium13, @victorrmclarity, and @DaniSancas for the contribution! 👍